frontend/src/pages/project/AccessControlPage/components/MembersTab/components/AddMemberModal.tsx

```diff
@@ -263,6 +308,19 @@ export const AddMemberModal = ({ popUp, handlePopUpToggle }: Props) => {
                 </FormControl>
               )}
             />
+            {requesterEmail && projectInviteList.requesterStatus.isProjectUser && (
```

Comment:
thoughts on disabling the user selection input component when requesterEmail is defined? 

<img width="790" alt="image" src="https://github.com/user-attachments/assets/ffb03744-675f-4cb5-853c-d4517f5e0252" />

Reply:
Should we? I mean it's not like a bad idea right? It's just prefilled and if user feels like 

`Oh! I should add the other person as well` then they can just do it in this process